apiserver/tls-config.md

```diff
@@ -0,0 +1,362 @@
+---
+title: TLS-Config
+authors:
+  - "@deads2k"
+reviewers:
+  - "@ironcladlou"
+  - "@sttts"
+  - "@enj"
+approvers:
+  - "@ironcladlou"
+  - "@sttts"
+creation-date: 2019-09-17
+last-updated: 2019-09-17
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# TLS Config for Externally Facing Services
+
+Cluster-admins need to be able to choose which cipher suites their kube-apiserver, oauth-server, and routers use.
+This is a balance of corporate policy and compatibility with existing systems.  This should be a shared, top level
+configuration amongst the affected components with a local override for routers to allow workloads to have a different 
+level of TLS config restrictions than the infrastructure.
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+Cluster-admins need to be able to choose which cipher suites their kube-apiserver, oauth-server, and routers use.
+This is a balance of corporate policy and compatibility with existing systems.  This should be a shared, top level
+configuration amongst the affected components with a local override for routers to allow workloads to have a different 
+level of TLS config restrictions than the infrastructure.
```

Comment:
Is it necessary to restate the point from the TLS Config for Externally Facing Services section here?